.gitignore

```diff
@@ -4,3 +4,4 @@ vendor
 composer.lock
 .php_cs.cache
 tests/_run
+/.idea
```

Comment:
I think it is generally better to add project irrelevant files to global (user) gitignore file even though most of the developer users use jetbrains IDEs